unabridged-works-volume-4/original/17-letters.adoc

```diff
@@ -392,11 +420,15 @@ but mayst meet with the right key, and the right understanding,
 wherein thy soul will be safe and happy; which is my heart`'s desire for thee,
 who am a real friend and well-wisher to thy soul in the Lord.
 
+[.signed-section-signature]
 I+++.+++ P.
 
-=== TO THE POOR AMONG FRIENDS, who are relieved by the charity and bowels of love, which God opens in other Friends towards them
+[.letter-heading]
```

Comment:
I wasn't sure if this should be a letter heading or blurb!

Reply:
yeah, tough... but i think you did the right thing.